src/components/organisms/about/about.scss

```
@@ -0,0 +1,35 @@
+@use "../../variables" as *;
+
+.about {
+  padding: 100px 0px;
+
+  @media (max-width: $breakpoint-lg) {
+    padding: 40px 0px;
+  }
+
+  .about-image {
+    width: 100%;
+    margin: 0;
+
+    img {
+      padding: 0;
+    }
+  }
+
+  #message2 {

```

```suggestion
  .message {
```
こちらも可能であればクラスセレクタを使いましょう！
`message.scss` との衝突を気にしているのであれば、今回の書き方では問題ないです。
上記のように修正した結果、生成される css は `.about .message` となり、詳細度の関係でこちらが優先されるので。
参考：https://developer.mozilla.org/ja/docs/Web/CSS/Specificity